src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -57,18 +68,23 @@
   </ol>
 
   <stache-page-anchor>
-    Installation for iOS Simulator on Mac
+    Manual installation for iOS Simulator on Mac
   </stache-page-anchor>
 
   <ol>
+    <li>
+      <sky-code-block>
```

Comment:
Same note above about the clunkiness of the code block in the step. It seems like it should just be a step that says "Run `skyux certs generate`."

Reply:
Same answer as above.  :-)